persistent-sqlite/Database/Persist/Sqlite.hs

```diff
@@ -539,7 +541,7 @@ getCopyTable :: [EntityDef]
              -> EntityDef
              -> IO [(Bool, Text)]
 getCopyTable allDefs getter def = do
-    stmt <- getter $ T.concat [ "PRAGMA table_info(", escapeE table, ")" ]
+    stmt <- getter $ T.concat [ "PRAGMA ", tableInfo, "(", escapeE table, ")" ]
```

Comment:
Again, there is one `table_info` per database, and you need to feed it an unqualified table name, instead of the qualified `foo.table`.